src/Raven.Server/Documents/Operations/Operations.cs

```diff
@@ -189,6 +189,9 @@ void ProgressNotification(IOperationProgress progress)
 
             _active.TryAdd(id, operation);
 
+            if (token == null)
```

Comment:
what kind of operation doesn't have a token?

Reply:
TransactionRecorder and generation of certificates